avc/src/planner/planner.cpp

```diff
@@ -1,129 +1,188 @@
 #include "planner.h"
-#include <cmath>
+
+using namespace std;
 
 planner::planner() {
-	ros::NodeHandle nh;
-	ros::NodeHandle pnh("~");
-
-	pnh.getParam("speed_increment", SPEED_INCREMENT);
-	pnh.getParam("angle_increment", ANGLE_INCREMENT);
-	pnh.getParam("time_increment", TIME_INCREMENT);
-	pnh.getParam("search_radius", SEARCH_RADIUS);
-	pnh.getParam("distance_increment", DISTANCE_INCREMENT);
-	map_sub = nh.subscribe("map", 1, &planner::mapCb, this);
-	speed_pub = nh.advertise<rr_platform::speed>("speed", 1);
-	steer_pub = nh.advertise<rr_platform::steering>("steering", 1);
-	path_pub = nh.advertise<nav_msgs::Path>("path", 1);
+    ros::NodeHandle nh;
+    ros::NodeHandle pnh("~");
+
+    pnh.getParam("steer_stddev", STEER_STDDEV);
+    pnh.getParam("max_steer_angle", MAX_STEER_ANGLE);
+    pnh.getParam("max_speed", MAX_SPEED);
+    pnh.getParam("path_stage_time", PATH_STAGE_TIME);
+    pnh.getParam("path_stages", PATH_STAGES);
+    pnh.getParam("collision_radius", COLLISION_RADIUS);
+    pnh.getParam("path_iterations", PATH_ITERATIONS);
+    pnh.getParam("time_increment", TIME_INCREMENT);
+
+    map_sub = nh.subscribe("map", 1, &planner::mapCb, this);
+    speed_pub = nh.advertise<rr_platform::speed>("speed", 1);
+    steer_pub = nh.advertise<rr_platform::steering>("steering", 1);
+    path_pub = nh.advertise<nav_msgs::Path>("path", 1);
+
+    normal_distribution<double> nd(0, STEER_STDDEV);
+    steering_gaussian = nd;
+    mt19937 rand_engine(std::random_device{}());
+    rand_gen = rand_engine;
```

Comment:
These initializations don't need to be broken into multiple lines.
You can just write `steering_gaussian = normal_distribution<double>(0, STEER_STDDEV);`